index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Mobile Version</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/a3673060fd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <section class="hero">
+        <header>
+            <div class="header-content flex-all">
+                <div class="logo">Welcome</div>
+                <ul class="nav-list">
+                    <li><a href="#">Portoflio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+                <div class="burger">
+                    <i class="fa-solid fa-bars"></i>
+                </div>
+            </div>
+        </header>
+        <main>
+            <div class="contant flex-center">
+                <h1>Hey there. I’m Ansar.</h1>
+                <h2>I’m a software developer</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <ul class="social flex-all">
+                    <li>
+                        <a href="#">
+                            <img src="assest/img/social/angellist.png" alt="angellist">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/AnsarIbrahim">
+                            <img src="assest/img/social/github.png" alt="github">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/ansar-ibrahim-61447424a/">
+                            <img src="assest/img/social/Linkedin icon.png" alt="Linkedin">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="assest/img/social/medium.png" alt="medium">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://twitter.com/ansaradheeb">
+                            <img src="assest/img/social/twitter.png" alt="twitter">
+                        </a>
+                    </li>
+                </ul>
+                <div class="next flex-all">
+                    <img src="assest/img/disabled/Disabled.png" alt="disabled">
+                </div>
+            </div>
+    </section>
+    <section class="works mt grid">
+        <h1 class="head mb">My Recent Works</h1>
```

Comment:
- [OPTIONAL]  It would be nice to change this to another `hx(h2-h6)` tag instead of the `h1` tag. Based on convention, there should be only one `h1` tag on a webpage. 💻